src/tls_wrap.cc

```diff
@@ -912,6 +932,15 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
 
 
+void TLSWrap::GetWriteQueueSize(const FunctionCallbackInfo<Value>& args) {
+  TLSWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
+
+  uint32_t write_queue_size = wrap->UpdateWriteQueueSize();
```

Comment:
Is it correct to call `UpdateWriteQueueSize()` here?  Since it updates the `.writeQueueSize` property, I think it would depend on the order of evaluation whether `handle.writeQueueSize === handle.getWriteQueueSize()`.

Reply:
Would the left side ever get evaluated after the right side? But also we store the `writeQueueSize` in a variable before comparing it.

```js
  const prevWriteQueueSize = this._handle.writeQueueSize;
  if (prevWriteQueueSize > 0 &&
      prevWriteQueueSize !== this._handle.getWriteQueueSize()) {
```

Ultimately, we need a way to update `writeQueueSize` here (for the next time _onTimeout runs) so I just wanted to avoid having to go in there twice.

Reply:
Then can you call it `.updateWriteQueueSize()`, to make it clear it has side effects?

> Would the left side ever get evaluated after the right side?

If you have to worry about such subtleties, I'd say it's time to, ah, _re-evaluate_ your code. :-)